src/test/regress/sql/pg15.sql

```diff
@@ -255,5 +255,89 @@ SELECT create_distributed_table('numeric_negative_scale','orig_value');
 
 SELECT * FROM numeric_negative_scale ORDER BY 1,2;
 
+--
+-- In PG15, unlogged sequences are supported
+-- we support this for distributed sequences as well
+--
+
+CREATE SEQUENCE seq1;
+CREATE UNLOGGED SEQUENCE seq2;
+
+-- first, test that sequence persistence is distributed correctly
+-- when the sequence is distributed
+
+SELECT relname,
+       CASE relpersistence
+            WHEN 'u' THEN 'unlogged'
+            WHEN 'p' then 'logged'
+            ELSE 'unknown'
+        END AS logged_info
+FROM pg_class WHERE relname = 'seq1' OR relname = 'seq2' ORDER BY relname;
```

Comment:
It might be a good idea to add a filter on the schema name to prevent other tests from using same sequence name in other schemas.

```suggestion
FROM pg_class
WHERE relname IN ('seq1', 'seq2') AND relnamespace='pg15'::regnamespace
ORDER BY relname;
```